tests/common/dhcp/test_dhcp.py

```diff
@@ -37,19 +42,12 @@ def test_wireserver_route_exists(self):
         self.assertTrue(dhcp_handler.gateway is None)
 
         # execute
-        routing_table = "\
-            Iface	Destination	Gateway 	Flags	RefCnt	Use	Metric	" \
-                        "Mask		MTU	Window	IRTT \n\
-            eth0	00000000	10813FA8	0003	0	    0	5	" \
-                        "00000000	0	0	0   \n\
-            eth0	00345B0A	00000000	0001	0	    0	5	" \
-                        "00000000	0	0	0   \n\
-            lo	    00000000	01345B0A	0003	0	    0	1	" \
-                        "00FCFFFF	0	0	0   \n"
+        routing_table = TestDHCP.DEFAULT_ROUTING_TABLE + \
```

Comment:
Could you rename so it's clear what this is for, something like `routing_table_with_wireserver_route`. Thanks!

Reply:
sure, that makes sense.